pkg/capabilities/v2/protoc/pkg/templates/server.go.tmpl

```diff
@@ -118,6 +124,11 @@ func (c *{{.GoName|LowerFirst}}Capability) RegisterTrigger(ctx context.Context,
                 input := &{{name .Input.GoIdent ""}}{}
                 return capabilities.RegisterTrigger(ctx, "{{(CapabilityId $service)}}", request, input, c.{{$service.GoName}}Capability.Register{{.GoName}})
         {{- end }}
+        {{- end }}
+        {{- if (SupportsLegacyTriggerApi $service) }}
+            case "":
+                // The capability supports the legacy trigger API, in the case of an empty method name invoke the legacy trigger API
+                return c.CronCapability.RegisterLegacyTrigger(ctx, request)
```

Comment:
See my prior comment, then we don't hard code this for cron only. I'm also hoping that we can re-use the callback we already have for Trigger and use the same type (I'm assuming no workflow uses the actual execution time).